v0/destinations/hs/data/HSTrackConfig.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "destKey": "utk",
+    "sourceKeys": ["traits.utk", "context.traits.utk", "properties.utk"],
```

Comment:
What is this field?

Reply:
User Token (Non required field) - Ref - https://developers.hubspot.com/docs/api/analytics/events